ingestion_server/ingestion_server/indexer.py

```diff
@@ -286,8 +274,8 @@ def replicate(self, table, dest_index, query):
                 es_batch = self.pg_chunk_to_es(
                     pg_chunk=chunk,
                     columns=server_cur.description,
-                    origin_table=table,
-                    dest_index=dest_index,
+                    model_name=model_name,
```

Comment:
Same question as above, should we pass `table_name` here instead?

Reply:
`pg_chunk_to_es` does not need the table name (it already has the chunk of rows from the table) but rather the model name so that it can map those rows to the right ES fields. This is correct and works as expected.

Reply:
This makes much more sense now that `database_table_to_elasticsearch_model` has been renamed! Got it, thank you :)